docs/quickstart-ui.md

```diff
@@ -0,0 +1,124 @@
+
+![Add Rancher Manager Extensions repository](images/quickstart-ui-extension-repository.png)
+
+[next section](#install-the-elemental-plugin)
+
+## Install the elemental plugin
+
+After the Rancher Manager Extensions Support is enabled, you can install the `elemental` plugin as follow:
+
+* Under the `Available` tab you will see `elemental` plugin available
+
+![Rancher Manager Available plugins](images/quickstart-ui-extensions-available.png)
+
+:::note
+If the `Available` tab shows no entries, refresh the page. The `elemental` plugin will then appear.
+:::
+
+* Click on the `Install` button, a popup will appear and click on `Install` again to continue.
+
+![Elemental plugin install](images/quickstart-ui-elemental-plugin-install.png)
+
+* On the `Installed` tab, the `elemental` plugin is now listed.
+
+:::note
+If the `elemental` plugin is listed and the status stays at `Installing...`, refresh the page. The `elemental` plugin will display correctly.
+:::
+
+Once the `elemental` plugin installed, you can see the `OS Manamagent` option in the Rancher Manager menu.
+
+![Rancher Manager OS Management menu](images/quickstart-ui-elemental-plugin-menu.png)
+
+## Add a Machine Registration Endpoint
+
+In the OS Management dashboard, click the `Create Registration Endpoint` button.
+
+![OS Management registration endpoints](images/quickstart-ui-registration-endpoint-create.png)
+
+Now here either you can enter each detail in its respective places or you can edit this as YAML and create the endpoint in one go. Here we'll edit this as YAML.
+
+```yaml showLineNumbers
+apiVersion: elemental.cattle.io/v1beta1
+kind: MachineRegistration
+metadata:
+  name: elemental-cluster1
+  namespace: fleet-default
+spec:
+  config:
+    cloud-config:
+      users:
+        - name: root
+          passwd: root
+    elemental:
+      install:
+        reboot: true
+        device: /dev/sda
+        debug: true
+      registration:
+        emulate-tpm: true
+        emulated-tpm-seed: 1
+  machineName:
+  machineInventoryLabels:
+    manufacturer: "${System Information/Manufacturer}"
+    productName: "${System Information/Product Name}"
+    serialNumber: "${System Information/Serial Number}"
+    machineUUID: "${System Information/UUID}"
+```
+
+Remove the existing content and enter this code snippet and click on save. Remember to modify this as per your need. Click the `Create` button.
+
+![Create a Registration Endpoint with YAML](images/quickstart-ui-registration-endpoint-create-yaml.png)
+
+:::info main options
+`name: elemental-cluster1`: change this as per your need
+
+`device: /dev/sda`: make sure your target device is "sda". Otherwise find out how the disk device is named and change it here. For example, in Raspberry Pi it could be "mmblk"
+
+`emulate-tpm: true`: use this only if your target device doesn't have a TPM device and you have a way of emulating TPM like in VMware or KVM
```

Comment:
nit: I would not use tpm emulation in the UI quickstart... ok to remove it later (we should also update the MachineRegistration image)

Reply:
Will be covered in a follow-up PR